commands/import.go

```diff
@@ -332,25 +333,93 @@ func importJobListCmdF(c client.Client, command *cobra.Command, args []string) e
 }
 
 type Statistics struct {
-	Schemes        int `json:"schemes"`
-	Teams          int `json:"teams"`
-	Channels       int `json:"channels"`
-	Users          int `json:"users"`
-	Emojis         int `json:"emojis"`
-	Posts          int `json:"posts"`
-	DirectChannels int `json:"direct_channels"`
-	DirectPosts    int `json:"direct_posts"`
-	Attachments    int `json:"attachments"`
+	Schemes        uint64 `json:"schemes"`
+	Teams          uint64 `json:"teams"`
+	Channels       uint64 `json:"channels"`
+	Users          uint64 `json:"users"`
+	Emojis         uint64 `json:"emojis"`
+	Posts          uint64 `json:"posts"`
+	DirectChannels uint64 `json:"direct_channels"`
+	DirectPosts    uint64 `json:"direct_posts"`
+	Attachments    uint64 `json:"attachments"`
 }
 
 func importValidateCmdF(command *cobra.Command, args []string) error {
 	configurePrinter()
 	defer printer.Print("Validation complete\n")
 
+	var (
+		serverTeams    = make(map[string]*model.Team) // initialize it in case we need to add teams manually
+		serverChannels map[importer.ChannelTeam]*model.Channel
```

Comment:
could we just use an id for the key?
```suggestion
		serverChannels map[string]*model.Channel
```

Reply:
We could use `teamID/channelID` or something like that. I'm using it here because it's used in the rest of the verification.